### PR TITLE
Refactor benchmarker

### PR DESCRIPTION
**Pull request by GorogPeter**
- unify names (e.g. "engine" insted of "e")
- unify naming convention (snake_case everywhere)
- unify string markers (double quotation mark everywhere)
- fstring (instead of string concatenation)
- remove default values (they were unused)
- raise error insted of terminating

Moreover, 2 typos have been fixed in FloatConvInl.h.

Signed-off-by: Görög Péter Sándor gorogpetersandor@gmail.com

### CODE REVIEW

File: test/wasmBenchmarker/benchmark.py [hunk 1]
```diff
@@ -24,7 +24,10 @@
 from os.path import abspath, dirname, join
 from markdownTable import markdownTable  # pip install py-markdown-table
 
-TEST_DIR = join(dirname(abspath(__file__)), 'ctests')
+
+TEST_DIR = join(dirname(abspath(__file__)), "ctests")
```
Comment (zherczeg) [inline test/wasmBenchmarker/benchmark.py:28]: This line did not change. Is this a newline issue?

File: test/wasmBenchmarker/benchmark.py [hunk 2]
```diff
@@ -140,135 +142,133 @@ def compile_tests(emcc_path, path, only_game=False, only_simd=False, compile_any
...
             continue
 
-        extraFlags = "-msimd128" if file.startswith("simd") else ""
-
-        if (verbose): print("compiling " + name)
-        bob_the_stringbuilder = emcc_path + " " + path + "/" + file + " " + extraFlags + " -s WASM=1 -s EXPORTED_FUNCTIONS=_runtime -s EXPORTED_RUNTIME_METHODS=ccall,cwrap -o " + path + "/wasm/" + name + ".wasm"
-        if verbose: print(bob_the_stringbuilder)
-        os.system(bob_the_stringbuilder)
+        flags = ("-msimd128" if file.startswith("simd") else "") \
+              + " -s WASM=1 -s EXPORTED_FUNCTIONS=_runtime"      \
```
Comment (zherczeg) [inline test/wasmBenchmarker/benchmark.py:150]: Do you need \ if you put the + before the next line? Is this a common style?

**Review by zherczeg — COMMENTED** [review]
Nice patch! Only a few changes.

File: test/wasmBenchmarker/benchmark.py (hunk 1 above)
Comment (zherczeg) [reply test/wasmBenchmarker/benchmark.py:28]: No. I haven't seen the ' to " change.

File: test/wasmBenchmarker/benchmark.py (hunk 2 above)
Comment (GorogPeter) [reply test/wasmBenchmarker/benchmark.py:150]: I've found these two:
```python
long_string = "This is a very long string " + \
              "that spans multiple lines " + \
              "but appears as a single line."
```
```python
long_string = ("This is a very long string "
               "that spans multiple lines "
               "but appears as a single line.")
```

In the first case, the place of **+** is personal preference, in my humble opinion.

Comment (zherczeg) [reply test/wasmBenchmarker/benchmark.py:150]: The second looks the best. There is no \ or +

Comment (GorogPeter) [reply test/wasmBenchmarker/benchmark.py:150]: Okay. I'll change it.

**Review by zherczeg — APPROVED** [review]
LGTM

**Review by clover2123 — APPROVED** [review]

**Pull request merged**